shared/chat/conversation/attachment-fullscreen/container.tsx

```diff
@@ -63,14 +49,15 @@ const Connected = Container.connect(
     )
     return {
       autoPlay: stateProps.autoPlay,
-      hotkeys: ['left', 'right'],
```

Comment:
we shouldn't have our connectors make these super specific props for these hocs (esp. desktop only and not in the actual prop def)